examples/ASP.NET Core/README.md

```diff
@@ -0,0 +1,118 @@
+# OpenAI ASP.NET Core Example
+
+This example demonstrates how to use the OpenAI .NET client library with ASP.NET Core's dependency injection container, registering a ChatClient as a singleton for optimal performance and resource usage.
```

Comment:
We may want to consider linking to things for DI and ASP.NET Core and such.   Probably better to go with an "Additional Resources" area at the bottom than inline links, but your call.